filters/include/pcl/filters/model_outlier_removal.h

```diff
@@ -48,20 +48,26 @@ namespace pcl
 {
   /** \brief @b ModelOutlierRemoval filters points in a cloud based on the distance between model and point.
    * \details Iterates through the entire input once, automatically filtering non-finite points and the points outside
-   * the model specified by setSampleConsensusModelPointer() and the threshold specified by setThreholdFunctionPointer().
    * <br><br>
    * Usage example:
    * \code
+   #include <pcl/filters/filter_indices.h>
+
+
    * pcl::ModelCoefficients model_coeff;
    * model_coeff.values.resize(4);
-   * model_coeff.values[0] = 0; model_coeff.values[1] = 0; model_coeff.values[2] = 1.5; model_coeff.values[3] = 0.5;
+   * model_coeff.values[0] = 0; 
+   * model_coeff.values[1] = 0; 
+   * model_coeff.values[2] = 1.5; 
```

Comment:
```suggestion
   * model_coeff.values[2] = 1; 
```
The first three coefficients of a plane model are the plane normal, which should have length 1